foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/FilterOperator.java

```diff
@@ -22,24 +22,24 @@
  */
 public interface FilterOperator {
     //All
-    public static final String equals = "equals";
-    public static final String not_equals = "not_equals";
-    public static final String any = "any";
-    public static final String in = "in";
-    public static final String exists = "exists";
-    public static final String missing = "missing";
+    String equals = "equals";
```

Comment:
Ah..  Missed the interface...
On 03-Jan-2016 12:43 am, "Rishabh" <notifications@github.com> wrote:

> In
> foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/FilterOperator.java
> <https://github.com/Flipkart/foxtrot/pull/104#discussion_r48686696>:
>
> > @@ -22,24 +22,24 @@
> >   */
> >  public interface FilterOperator {
> >      //All
> > -    public static final String equals = "equals";
> > -    public static final String not_equals = "not_equals";
> > -    public static final String any = "any";
> > -    public static final String in = "in";
> > -    public static final String exists = "exists";
> > -    public static final String missing = "missing";
> > +    String equals = "equals";
>
> Interface variables are by default static and final.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Flipkart/foxtrot/pull/104/files#r48686696>.
>